src/Tests/Finos.Fdc3.NewtonsoftJson.Tests/Context/OrganizationTests.cs

```diff
@@ -19,12 +19,12 @@ namespace Finos.Fdc3.NewtonsoftJson.Tests.Context;
 public class OrganizationTests : ContextSchemaTest
 {
     public OrganizationTests()
-        : base("https://fdc3.finos.org/schemas/2.0/organization.schema.json")
+        : base("https://fdc3.finos.org/schemas/2.1/context/organization.schema.json")
     {
     }
 
     [Fact]
-    public async void Organization_SerializedJsonMatchesSchema()
+    public async Task Organization_SerializedJsonMatchesSchema()
     {
         Organization organization = new Organization(new OrganizationID() { FDS_ID = "fdc_id", LEI = "lei", PERMID = "permid" }, "organization");
```

Comment:
Very minor (and outside the changes, but since I noticed), it seems to be "fdcsid" everywhere else, so I was going to suggest  changing for consistency
```suggestion
        Organization organization = new Organization(new OrganizationID() { FDS_ID = "fdsid", LEI = "lei", PERMID = "permid" }, "organization");
```

Reply:
Looks like maybe a copy/paste and not changing all mock test data.  I'll go through and map the mock data to the field name for all instances of this field.

Reply:
Updated